test/Interop/SwiftToCxx/functions/swift-primitive-inout-functions-cxx-bridging.swift

```diff
@@ -0,0 +1,29 @@
+// RUN: %empty-directory(%t)
+// RUN: %target-swift-frontend %s -typecheck -module-name Functions -clang-header-expose-public-decls -emit-clang-header-path %t/functions.h
+// RUN: %FileCheck %s < %t/functions.h
+
+// RUN: %check-interop-cxx-header-in-clang(%t/functions.h)
+
+// CHECK:      inline void inOutInt(swift::Int & x) noexcept {
```

Comment:
Can you also add a `CHECK` line to test that the generated C declaration that you're calling inside this stub has the `*` type?